src/Android/Xamarin.Android/Samples/GeometryEngine/BufferList/BufferList.cs

```diff
@@ -123,7 +123,7 @@ private void MyMapView_GeoViewTapped(object sender, GeoViewInputEventArgs e)
             {
                 // Display an error message if there is a problem generating the buffer polygon.
                 AlertDialog.Builder alertBuilder = new AlertDialog.Builder(this);
-                alertBuilder.SetTitle("Geometry Engine Failed!");
+                alertBuilder.SetTitle("Geometry Engine failed!");
```

Comment:
[sorry, I know this isn't part of the PR, feel free to ignore]
I think we should phrase this message differently and lose the exclamation (it sounds pretty harsh). It's also not necessarily accurate, I'm sure there are other reasons why this may fail. 
How about simply "Unable to generate buffers"?